src/main/java/org/repodriller/filter/diff/DiffFilter.java

```diff
@@ -0,0 +1,17 @@
+package org.repodriller.filter.diff;
```

Comment:
The idea of a `DiffFilter` seems like it should offer the user more to work with than the file name. Should we change the name, or offer the user the commit ID?

Reply:
What do we achieve by offering the commit ID? The commit has multiple Diffs according to this model, with each Diff being associated with a file that was modified.

Reply:
At the moment a `DiffFilter` can *only* use the file name to make its decision. This seems unnecessarily restrictive. Would it be possible to provide more information?

Reply:
Ideally we would provide a `DiffEntry` or something similar. Then the user could define filters based on data about diff like size, etc. But that makes it Git-specific.

An alternative is to create a lightweight VCS-agnostic `Diff` object that encodes anything common we can abstract out of `DiffEntry` and whatever's Subversion's `DiffEntry` is, and pass that around. So this could theoretically encode *at least* the fileName and the size of the diff, and maybe some other stuff as well.

Thoughts? @mauricioaniche?

Reply:
I like the `DiffEntry` idea.

Reply:
Yes, I suppose this has to be at file level.